plugins/carlabase/carla.cpp

```diff
@@ -33,13 +33,20 @@
 #include "InstrumentPlayHandle.h"
 #include "InstrumentTrack.h"
 #include "Mixer.h"
+#include "Knob.h"
+#include "EffectControls.h"
```

Comment:
I think you will not need to include `EffectControls.h` here as well as in `carla.h`.